### ISSUE TITLE
Interaction between ODIS/Unesco with TDWG and DISSCO on Schema.org mapping to TDWG standards 

### ISSUE THREAD

**Issue by pmergen** [iodepo/odis-arch#434]
Based on a discussing of addressing thematic siloing, would need to interact with TDWG and their biodiversity standards, Schema.org as well as with DiSSCo to open in practice to other platforms and other disciplines. Plans would to test with OBIS/UNESCO . Link it also to the reviving of the RDA Biodiverity group to add OBIS to it. 
involved: 
Pier Luigi Buttigieg
Sam Leeflang
Sharif Islam
Patricia Mergen

back ground reading : https://worldfair-project.eu/2023/12/05/worldfair-new-interoperability-specifications-and-policy-recommendations-d11-2/

**Comment by jmckenna** [comment]
thanks @pmergen.  Did you mean above "Plans would be to test with ODIS/UNESCO" ?  (I wasn't sure if you swapped ODIS with OBIS in that sentence)

**Comment by pmergen** [comment]
in the discussion ODIS and OBIS was mentionned ..

**Comment by pbuttigieg** [comment]
@jmckenna @pmergen the aim here is for information standards organisations like TDWG to offer authoritative maps between the standards they operate (e.g. TDWG's DwC and schema.org) 

We set an example of a machine-actionable map using SSSOM between DwC and the Genomic Standards Consortium's MIxS standards here https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10565567/

If TDWG can maintain such a map to schema.org, DCAT, and other generic standards (naturally, only covering the generic content), then systems like ODIS could steam ahead with biodiversity data harmonisation when it interacts with systems like DiSSCo

**Comment by sharifX** [comment]
@pmergen @pbuttigieg

Thanks for initiating this discussion. From DiSSCo's and the overall biodiversity data integration and FAIR points of view, we are very much interested in this. We are also involved in some of the mapping and SSSOM discussions, so that alignment is beneficial (see a recent [hackathon around SSSOM](https://github.com/bge-barcoding/StayingMapped) in the Biodiversity Genomics Europe project). There is also ongoing discussion in the [Biodiversity Digital Twin project](https://github.com/BioDT/biodt-fair) around this topic.

To give a quick summary, DiSSCo is currently working on an MVP, and we are actively developing our Digital Specimen data model (based on FAIR Digital Object specifications). You can see our schema repository [here](https://schemas.dissco.tech/schemas/) and on [GitHub](https://github.com/DiSSCo/openDS).

Here's a simple high level mapping example that could be easily done. We can enhance this with SSSOM structure.

openDS (ods) -> scehma.org

- `ods:id` -> `identifier` (https://schema.org/identifier)
- `ods:version` -> `version` (https://schema.org/version)
- `ods:created` -> `dateCreated` (https://schema.org/dateCreated)
- `ods:specimenName` -> `name` (https://schema.org/name) _Note: this needs to allow both biological and geological specimen names._
- `dcterms:license` -> `license` (https://schema.org/license)
- `dcterms:modified` -> `dateModified` (https://schema.org/dateModified)
- `dcterms:accessRights` -> `hasDigitalDocumentPermission ` (https://schema.org/hasDigitalDocumentPermission or https://schema.org/conditionsOfAccess)
- `dcterms:rightsHolder` -> `accountablePerson` (https://schema.org/accountablePerson)
- `dwc:institutionId`, `dwc:institutionName` -> `provider` (https://schema.org/provider)
- `dwc:recordedBy` -> `creator` (https://schema.org/creator)

**However:**

Properties such as `ods:midsLevel`,  `ods:physicalSpecimenId`, `ods:physicalSpecimenIdType`, `dwc:preparations`, `dwc:disposition`, `dwc:collectionCode`, `dwc:collectionId`, `dwc:informationWithheld`, and `dwc:verbatimLabel` do not have direct equivalents in schema.org. 

We are currently working on a more formalised and structured description of these terms to make them more FAIR. As some of these terms are related to Digital Specimen, specimen digitisation pipeline, and museum workflow, and are specific to DiSSCo, we will probably need to maintain the namespace for them. However, we can align with TDWG around the mapping schema.

And yes, syntactic variation in the value space is always challenging.

If this can be achieved, that would be amazing:

> "If TDWG can maintain such a map to schema.org, DCAT, and other generic standards (naturally, only covering the generic content), then systems like ODIS could steam ahead with biodiversity data harmonization when it interacts with systems like DiSSCo."


We also need to reach out to various other projects discussing mapping, like FAIRCORE4EOSC's [metadata schema cross walk registry](https://faircore4eosc.eu/eosc-core-components/metadata-schema-and-crosswalk-registry-mscr).

**Comment by pmergen** [comment]
Input from Ben Norton :

Two short notes to append.
1. Latimer Core is already mapped to [schema.org](http://schema.org/).
2. Schema.org is a 'high-level' schema for generally applicable concepts. Several groups within the [schema.org](http://schema.org/) space are actively extending the schema to more targeted areas, such as Taxon (https://schema.org/Taxon), but it remains more generalized than most things under the TDWG umbrella. Since most concepts will be narrower matches to terms in [schema.org](http://schema.org/), I think it's questionable if there are significant interoperability advantages to spending significant resources on this topic. Regardless, mapping to [schema.org](http://schema.org/) is a relatively straightforward process. I don't think it requires the establishment of a group; rather, someone just needs to do it.

**Comment by wouteraddink** [comment]
@pmergen can you add me too?

**Comment by pmergen** [comment]
Yes. It is an open GitHub . I think you cab access it and answer to the
tread.

I tried to add Ben but his Github name would not appear in the list..

Pat
